apps/test/netsim/NetSimRouterNode.js

```diff
@@ -252,6 +302,12 @@ describe("NetSimRouterNode", function () {
       // Here, the payload gets 'cleaned' down to empty string, then treated
       // as zero when parsing the toAddress.
       NetSimMessage.send(testShard, from, to, 'garbage', function () {});
+
+      // Router must tick to process messages; 1000ms is sufficient time for
+      // a short packet.
+      router.tick({time: 0});
+      router.tick({time: 1000});
```

Comment:
Just realized that in these cases I should reduce this to one tick; `tick({time: 0})` should be a `beforeEach` step.